### DISCUSSION TITLE
🐘 BOAZ MLOps 소모임에 오신 것을 환영합니다! 🐘

### DISCUSSION THREAD

**Discussion by Zerohertz** [BOAZ-Ops/.github#3] [Announcements]
1. 처음 가입 시 [이 링크](https://github.com/BOAZ-Ops/Introduce)에서 [양식](https://github.com/BOAZ-Ops/Introduce#template)에 맞춰 본인 소개를 해주세요!
2. 소모임의 활동은 [여기](https://github.com/orgs/BOAZ-Ops/projects)에서 확인할 수 있어요!

---

+ 문의 사항은 다른 사람들도 함께 공유할 수 있게 [여기](https://github.com/orgs/BOAZ-Ops/discussions/categories/q-a)에 남겨주세요!

**Comment by Zerohertz** [comment]
2023/07/12 공지

![image](https://github.com/BOAZ-Ops/.github/assets/42334717/e661617f-df92-435a-8d03-6fe7fa3fd053)
---
@NayeonKeum: 깃허브 사용에 어려움을 느끼신다면 말씀하시면 도와드리겠습니다!
---
@tae2089: 
![image](https://github.com/BOAZ-Ops/.github/assets/42334717/dc240223-968c-4c21-b95b-1c688457c78a)
관심 있으신 분들 선착순으로 2명 까지 받을 예정입니다! 연락 부탁드려요~
---
@Zerohertz:
![image](https://github.com/BOAZ-Ops/.github/assets/42334717/87b648b3-73f9-4fb7-a2a0-c7896920b284)

**Reply by NayeonKeum** [reply]
짤 하나 올렸다고 공개 샤라웃은 너무한 거 아닙니까.... 저도 뉴비와 다를 바가 없는디요..
